portalnet/src/storage.rs

```diff
@@ -285,6 +285,13 @@ impl PortalStorage {
         // Set the metrics to the default radius, to start
         storage.metrics.report_radius(storage.radius);
 
+        // Report total storage used by network content.
+        let network_content_storage_usage =
+            storage.get_total_storage_usage_in_bytes_from_network()?;
+        storage
+            .metrics
+            .report_content_data_storage(network_content_storage_usage as f64 / 1000.0);
```

Comment:
unrelated, but it would add clarity for the `report_` methods to have a unit suffix (e.g. kb) where the expected unit is not obvious

Reply:
Maybe this can go into the PR to switch to MB in #634 -- it seems silly to add kb now just for @mrferris to churn it to MB in a day or two :)